packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -281,6 +282,15 @@ const buildJupyterContext = () => {
 };
 
 const inputChangeHandler = async () => {
+	let modelId = '';
+	if (props.node.inputs[0]?.type === 'modelId') {
+		modelId = props.node.inputs[0].value?.[0];
+	} else if (props.node.inputs[0]?.type === 'modelConfigId') {
+		const modelConfiguration = await getModelConfigurationById(props.node.inputs[0].value?.[0]);
+		if (modelConfiguration) {
+			modelId = modelConfiguration.model_id;
+		}
+	}
```

Comment:
Can we not have so much repetition? It generally makes the code cleaner, easier to maintain and easier to read.

```
const input = props.node.input[0];
if (!input) return;

... stuff ...
```

Likewise in `tera-mira-stratify.vue`